flask_opentracing/tracing.py

```diff
@@ -162,6 +164,22 @@ def _after_request_fn(self, response=None, error=None):
 
         if response is not None:
             scope.span.set_tag(tags.HTTP_STATUS_CODE, response.status_code)
+
+            if self._trace_response_header:
+                expose_headers = response.headers.get(
+                    "Access-Control-Expose-Headers", ""
+                )
+                if expose_headers:
+                    expose_headers += ","
+                expose_headers += "Server-Timing"
+                response.headers["Access-Control-Expose-Headers"] = expose_headers
```

Comment:
I think it is worth to add an assertion if unit test to ensure that `Server-Timing` is added to `Access-Control-Expose-Headers` HTTP Header. Especially that there is logic here with the comma separation, 

Side comment. @johnbley have you checked that this is how `http.Header.Add` works in Go? I am not sure if I should double-check this implementation https://github.com/signalfx/signalfx-go-tracing/pull/34 or is it fine.